test/integration/patients/sidebar/flow-sidebar.js

```diff
@@ -739,8 +744,74 @@ context('flow sidebar', function() {
       .find('.patient-flow__name')
       .click();
 
+    cy
+      .intercept('PATCH', '/api/flows/1', {
+        statusCode: 204,
+        body: {},
+      })
+      .as('routePatchFlow');
+
     cy
       .get('[data-permission-region]')
       .should('contain', 'You are not able to change settings on this flow.');
   });
+
+  specify('flow with work:authored:delete permission', function() {
+    cy
+      .routesForPatientDashboard()
+      .routeCurrentClinician(fx => {
+        fx.data.relationships.role = { data: { id: '77777' } };
+        fx.data.relationships.team = { data: { id: '11111', type: 'teams' } };
+
+        return fx;
+      })
+      .routeFlow(fx => {
+        fx.data.id = '1';
+        fx.data.attributes.name = 'Owned by team member';
```

Comment:
Do we want to rename this one too?

Reply:
It's probably ok.. There's really no point in naming this one at all.